todo_app/templates/add_item.html

```diff
@@ -0,0 +1,64 @@
+<form action="/add-item" method="POST">
```

Comment:
Are these for experimenting? I can't see any evidence the new HTML files are actually used currently